.github/workflows/integration_test.yaml

```
@@ -84,17 +85,17 @@ jobs:
     needs:
       - collect-integration-tests
     runs-on: ${{ matrix.job.runner }}
-    timeout-minutes: 217  # Sum of steps `timeout-minutes` + 5
+    timeout-minutes: 217 # Sum of steps `timeout-minutes` + 5
     steps:
-      - name: Free up disk space
-        timeout-minutes: 1
-        run: |
-          printf '\nDisk usage before cleanup\n'
-          df --human-readable
-          # Based on https://github.com/actions/runner-images/issues/2840#issuecomment-790492173
-          rm -r /opt/hostedtoolcache/
-          printf '\nDisk usage after cleanup\n'
-          df --human-readable
+      # - name: Free up disk space
+      #   timeout-minutes: 1
+      #   run: |
+      #     printf '\nDisk usage before cleanup\n'
+      #     df --human-readable
+      #     # Based on https://github.com/actions/runner-images/issues/2840#issuecomment-790492173
+      #     rm -r /opt/hostedtoolcache/
+      #     printf '\nDisk usage after cleanup\n'
+      #     df --human-readable
```

```suggestion
      - name: Disk usage
        timeout-minutes: 1
        run: df --human-readable
```
from https://github.com/canonical/data-platform-workflows/blob/8d0d111687006fc6653b5b6548e9563bb217872d/.github/workflows/integration_test_charm.yaml#L223